File: Ad-Hoc/pagina33/3488/3488.py
class Binary_Hexadecimal_Number_Systems:
    @staticmethod
    def isPowerOfTwo(n: int) -> bool:
        return n > 0 and (n & (n - 1)) == 0
        
n = int(input())
ispot = Binary_Hexadecimal_Number_Systems.isPowerOfTwo(n)
print("true" if ispot else "false")

File: Ad-Hoc/__3000__/3488/3488.cpp
#include <iostream>
using namespace std;

class BinaryHexadecimalNumberSystem {
public:
    static bool isPowerOfTwo(int n) {
        return n > 0 && (n & (n - 1)) == 0;
    }
};

int main() {
    
    int n;
    cin >> n;
    
    bool isPot = BinaryHexadecimalNumberSystem::isPowerOfTwo(n);
    cout << (isPot == 1 ? "true" : "false") << endl;
    
    return 0;
}